mbs_results/data_cleaning.py

```diff
@@ -182,14 +182,63 @@ def load_manual_constructions(
     )
 
 
+def join_manual_constructions(
+    df: pd.DataFrame,
+    manual_constructions: pd.DataFrame,
+    reference: str,
+    period: str,
+    **config
+):
+    """
+    function to change datatypes of columns based on config file
+
+    Parameters
+    ----------
+    df: pd.DataFrame
+        dataframe with combined responses and contributors columns
+        index is expected to be 'period' and 'reference'
+    manual_constructions_path: str
+        the manual construction dataframe
+    reference: str
+        the name of the reference column
+    period: str
+        the name of the period column
+    **config: Dict
+        main pipeline configuration. Can be used to input the entire config dictionary
+
+    Returns
+    -------
+    pd.DataFrame
+        dataframe with correctly formatted column datatypes.
+    """
+    if manual_constructions[period].dtype != df[period].dtype:
+        manual_constructions[period] = convert_column_to_datetime(
+            manual_constructions[period]
+        )
+
+    if manual_constructions[reference].dtype != df[reference].dtype:
+        reference_dtype = df[reference].dtype
+        manual_constructions[reference] = manual_constructions[reference].astype(
+            reference_dtype
+        )
+
+    manual_constructions.set_index([reference, period], inplace=True)
+    df.set_index([reference, period], inplace=True)
+
+    validate_manual_constructions(df, manual_constructions)
+
+    return df.merge(
+        manual_constructions, on=[reference, period], how="outer", suffixes=("", "_man")
```

Comment:
Can we please add a comment why "outer" join ? I think it will be nice to have the motivation of this documented in the pull request.

Reply:
Excellent spot. I think it was coppied from elsewhere. Changed this to a left join (which is what I would expect to be used here) and unit tests still pass.